docs/pipelineruns.md

```diff
@@ -161,14 +161,31 @@ spec:
 **Note: This is only allowed if `enable-tekton-oci-bundles` is set to
```

Comment:
one thing I missed earlier! remote resolution syntax doesn't depend on the enable-oci-bundles feature flag, but this example makes it sound like it does. Same comment for bundles syntax used elsewhere in this PR.

Reply:
maybe something like:
```
You can reference a Tekton bundle in a TaskRef in both v1 and v1beta1 using remote resolution (link). The example syntax shown below for v1 uses remote resolution, and requires enabling xyz feature flag.

In v1beta1, you can also reference a Tekton bundle using OCI bundle syntax, which has been deprecated in favor of remote resolution. The example shown below for v1beta1 uses OCI bundle syntax, and requires enabling xyz feature flag.
```

Reply:
Thanks! Done.